### ISSUE TITLE
Run as daemon

### ISSUE THREAD

**Issue by davwilliams** [bewilliams/internet_availability_monitor#2]
The startup script is great, but it would be nice to install this as a daemon in the background and be able to start and stop it like any service.  I'll submit a PR for that later.

**Comment by bewilliams** [comment]
yeah, I've already thought of that. I just haven't written the script for that yet.

**Comment by bewilliams** [comment]
Another thing is that I would like for it to be user independent.

Currently, the scripts rely on things being in the /home/pi directory. This is great if you want to use that user (or more specifically already have that user on a raspberry pi), but if you want to move to another user it wont work yet.
